map: Нова дополнительные сабмапы и правки старых

PiroMage opened the pull request:
1) Добавлены сабмапы для старой библиотеки.
2) Добавлены сабмапы для старого медотдела.
3) Добавлены сабмапы для старой церкви
4) Изменены старые сабмапы, чтобы не конфликтовали зонами с другими.

<!-- **ВАЖНО!!!** Если Ваш ПР содержит много .dmi файлов, которые могут создать излишнюю нагрузку на IconDiffBot2, тогда добавьте в название ПРа тэг [IDB IGNORE] -->
<!-- Например, "Добавил 1kk новых спрайтов [IDB IGNORE]" -->

<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- Список тегов для ПРа:
- add: Вы добавляете что-то новое.
- admin: Вы меняете что-то связанное с администрацией. (кнопки, управление, панели, щитспавн и т.д.)
- balance: Вы производите балансировку в игре.
- bugfix: Вы исправили некий баг.
- code_imp: Вы имплементируете новое для билда, не меняя при этом ничего в самой игре.
- config: Вы меняете конфиг или работу SQL.
- del: Вы удаляете что-то из игры.
- experiment: Ваш ПР сделан с целью какого-то эксперимента.
- map Вы меняете только карту.
- local Вы делаете добавляете новый текст на русском или переводите на русский.
- imageadd:  Вы добавили новый спрайт.
- imagedel:  Вы удалили старый спрайт.
- soundadd: Вы добавили новый звук.
- sounddel: Вы удалили старый звук.
- spellcheck: Вы исправили опечатку.
- tweak: Вы внесли незначительную правку.
- refactor: Вы полностью переписали старый код, улучшив его, НО не изменив функционал.
- server: Вы меняете что-то связанное с серверной частью или Github.
- wip: Ваш PR в драфте и планируется длительная разработка. -->

## Что этот ПР делает

<!-- Опишите, что делает ваш пулл-реквест, укажите основные изменения. -->
<!-- В случае исправления бага, укажите ссылку на канал в #баг-репорты-v2 или issue в репозитории. В ином случае, опишите баг и ступени для его воспроизведения. -->

## Почему это хорошо для игры
Вариативность
<!-- Опишите, почему, по вашему, следует добавить эти изменения в игру. -->

## Демонстрация изменений

<!-- Заполните, если Вы меняли спрайты, карту или ваши изменения требуют визуальной демонстрации изменений. -->

## Тестирование
Тестировал на локалке
<!-- Как Вы тестировали свои изменения? Ведь тестировали? -->

PiroMage committed:
Новые сабмапы и правки старых

1) Добавлены сабмапы для старой библиотеки.
2) Добавлены сабмапы для старого медотдела.
3) Изменены старые сабмапы, чтобы не конфликтовали зонами с другими.

PiroMage committed:
Небольшие правки в карты

Den109G commented on _maps/map_files/nova/submaps/AbondonedMed.dmm:
```diff
@@ -0,0 +1,7593 @@
+/area/maintenance/medroom)
+"ft" = (
+/obj/machinery/door/poddoor/multi_tile/three_tile_hor{
+	id = "hangarshuttlebay4";
```
`id`, вроде как, нет у поддверей.

NightDawnFox committed:
фикс для пиро

NightDawnFox committed:
добавлены маркеры

NightDawnFox committed:
еще пара айди

Ravgolin commented:
<img width="89" height="77" alt="image" src="https://github.com/user-attachments/assets/52d92f06-babf-469e-a352-18608279cf58" />
<img width="123" height="127" alt="image" src="https://github.com/user-attachments/assets/0746f37c-28d2-4ca5-b697-eb5d95f5b4cd" />
<img width="144" height="147" alt="image" src="https://github.com/user-attachments/assets/a6e66fa6-0a8c-4e88-bfe6-ff6a2729efc8" />
Есть незначительные мелочи, грязь в стенах на сабмапе "East mantiance"

Ravgolin commented:
<img width="374" height="543" alt="image" src="https://github.com/user-attachments/assets/c5bcc2fb-4f4c-4409-95f7-a714e6f878ca" />
Ну блэкбоксы  в техах странная тема, лучше уж нефункциональные консолики.

PiroMage committed:
Небольшие правки

1) Глянул на картах грязь под стенами - убрал.
2) Поменял черные ящики в старой серверной на просто фреймы серверов.

PiroMage committed:
Merge branch 'Nova-submapsV2' of https://github.com/PiroMage/Paradise into Nova-submapsV2
